apps/liquidity-provision-dashboard/src/app/components/detail/Market/market.tsx

```diff
@@ -0,0 +1,164 @@
+import { useState } from 'react';
+import {
+  t,
+  useDataProvider,
+  useYesterday,
+  makeDerivedDataProvider,
+  formatNumber,
+} from '@vegaprotocol/react-helpers';
+import { Icon } from '@vegaprotocol/ui-toolkit';
+import {
+  formatWithAsset,
+  calcDayVolume,
+  getChange,
+  displayChange,
+} from '@vegaprotocol/liquidity';
+
+import {
+  MarketTradingModeMapping,
+  MarketTradingMode,
+  AuctionTrigger,
+  AuctionTriggerMapping,
+} from '@vegaprotocol/types';
+import { marketCandlesProvider } from '@vegaprotocol/market-list';
+import { Interval } from '@vegaprotocol/types';
+import { HealthBar } from '../../health-bar';
+import { HealthDialog } from '../../health-dialog';
+
+interface Levels {
+  fee: string;
+  commitmentAmount: number;
+}
+
+interface settlementAsset {
+  symbol?: string;
+  decimals?: number;
+}
+
+const candlesProvider = makeDerivedDataProvider(
```

Comment:
@mattrussell36 Please let me know if this should be done differently - the existing hook for a single market in `libs/liquidity` does not include candles, and I can't find any hooks for the candles. Both `apps/trading` and `apps/console-lite` imports the provider directly.

Reply:
Looks like this code has changed?

Reply:
Yes, this is now in `apps/liquidity-provision-dashboard/src/app/components/detail/last-24h-volume/last-24h-volume.tsx`